src/Signer/Key/InMemory.php

```diff
@@ -18,13 +19,21 @@ final class InMemory implements Key
 
     private function __construct(string $contents, string $passphrase)
```

Comment:
We should document `$contents` as `@param non-empty-string $contents`, then suppress any raised static analysis issues reported because of it, for now.